wiki/Ranking_Criteria/Difficulty_Naming/id.md

```diff
@@ -190,3 +190,4 @@ Nama tingkat kesulitan berikut adalah alternatif umum dan tambahan untuk nama ti
 - ![](/wiki/shared/diff/hard-o.png) Hyper: dalam spread Easy-Normal-Hard-Insane-Expert, yaitu tingkat kesulitan antara Hard dan Insane.
```

Comment:
add `Shokyuu` in favour of #6490?

Reply:
yes I will add it after #6490 get merged